Add option to display battery status as hearts in battery.sh

nikunjmathur08 opened the pull request:
#438 

This PR restores the "hearts" display for the `cute` battery mode, as mentioned in issue #438. It also introduces a new `symbol` option to display the battery status with a battery symbol.

xx4h commented:
Thanks a lot for your work! 

You're using `BATTERY_FULL` and `BATTERY_EMPTY` inside of `__generate_hearts()` which will just use the battery symbols and not the hearts. I suggest we bring back `HEART_FULL` and `HEART_EMPTY`, removed by this commit: https://github.com/erikw/tmux-powerline/commit/cab07470a7467068b6439a0cdd522be8c02e01fb

Besides that, I just have 3 small remarks:
1. Add the missing `;;` mentioned by the linter
2. Change `TMUX_POWERLINE_SEG_BATTERY_NUM_BATTERIES_DEFAULT` to `TMUX_POWERLINE_SEG_BATTERY_NUM_HEARTS_DEFAULT` in 
https://github.com/erikw/tmux-powerline/blob/ba893c98eedc3311cc4dca4a21387c59ec4ad5dd/segments/battery.sh#L5 and https://github.com/erikw/tmux-powerline/blob/ba893c98eedc3311cc4dca4a21387c59ec4ad5dd/segments/battery.sh#L18 so it matches the other variable and the use case.
3. In https://github.com/erikw/tmux-powerline/blob/ba893c98eedc3311cc4dca4a21387c59ec4ad5dd/segments/battery.sh#L15 add the new `hearts` option to the doc line?

nikunjmathur08 commented:
Made the following changes:

- Reintroduced HEART_FULL and HEART_EMPTY variables for displaying hearts.
- Renamed TMUX_POWERLINE_SEG_BATTERY_NUM_BATTERIES_DEFAULT to TMUX_POWERLINE_SEG_BATTERY_NUM_HEARTS_DEFAULT.
- Added the 'hearts' option to the TMUX_POWERLINE_SEG_BATTERY_TYPE documentation.
- Fixed the case statement by adding the missing ;; after the "hearts" case.

Fixes #438

nikunjmathur08 commented:
I've also removed the extra 7th line.

xx4h commented on segments/battery.sh:
```diff
@@ -39,6 +41,9 @@ run_segment() {
 		output=$(__cutinate "$battery_status")
 		;;
+	"hearts")
+		output=$(__generate_hearts "$battery_status")
```
```suggestion
		output=$(__generate_hearts "${battery_status/* /}")
```

While testing, i found out, that `$battery_status` is already containing a battery symbol. This is unfortunately baked into all the sub functions directly and needs a bigger rework. For now see my suggestion above.

xx4h commented:
LGTM, tested on linux.

Thanks again @nikunjmathur08!

nikunjmathur08 commented:
@xx4h can you please add this under Hacktoberfest?
